<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dept</class>
 <widget class="QDialog" name="Dept">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Список служб</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="3">
       <widget class="QTableView" name="deptsTableView"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>223</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Добавить службу</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="deptLongName"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="deptShortName"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Краткое наименование службы</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>264</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Оменить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Полное наименование службы</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>Ок</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dept</receiver>
   <slot>delDept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Dept</receiver>
   <slot>cancelAddDept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dept</receiver>
   <slot>addDept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Dept</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addDept()</slot>
  <slot>delDept()</slot>
  <slot>cancelAddDept()</slot>
 </slots>
</ui>
